cmd/account/console.go

```diff
@@ -123,7 +132,42 @@ func (o *consoleOptions) run() error {
 		fmt.Fprintln(o.Out, callerIdentityOutput)
 	}
 
+	// If BYOC we need to role-chain to use the right creds.
+	// Use the OrgAccess Role by default, override if BYOC
 	roleName := awsv1alpha1.AccountOperatorIAMRole
+
+	// TODO: Come back to this and do a lookup for the account CR if the account ID is the only one set so we can do this too.
+	if isBYOC {
+		// TODO: Change this second param to 'awsv1alpha1.SREAccessRoleName' once that gets merged
+		roleArn := fmt.Sprintf("arn:aws:iam::%s:role/%s", *callerIdentityOutput.Account, "RH-SRE-CCS-Access")
```

Comment:
For #66 I got this from the ConfigMap (since that is the source of truth from the a-a-o point of view)

Reply:
Hmm, originally I didn't get it from the configmap because I was thinking that this might be run independently of any of the hives, but the more that I think about it the less I think that will ever be the case, considering we need to query the hive to get the account CR, right?  That makes it a bit easier.

Reply:
Yeah, I think it should always query the a-a-o on hive since it needs the CR anyway. If we ever change the name of the role too, then we only need to change it in a-a-o